x-pack/plugins/watcher/server/lib/license_pre_routing_factory/license_pre_routing_factory.ts

```diff
@@ -14,7 +14,7 @@ import { RouteDependencies } from '../../types';
 
 export const licensePreRoutingFactory = (
   { getLicenseStatus }: RouteDependencies,
-  handler: RequestHandler
+  handler: RequestHandler<any, any, any>
```

Comment:
This is the only exception to defining types for handlers, it's the license route wrapper which doesn't affect the downstream types.